recipes/crowcpp-crow/all/conanfile.py

```diff
@@ -16,13 +17,15 @@ def _source_subfolder(self):
 
     def requirements(self):
         self.requires("boost/1.75.0")
-        '''To be removed in next released versions'''
+        # To be removed in next released versions.
         self.requires("openssl/1.1.1i")
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version])
-        extracted_dir = "crow-" + self.version
-        os.rename(extracted_dir, self._source_subfolder)
+        tools.get(
+            **self.conan_data["sources"][self.version],
+            strip_root=True,
+            destination=self._source_subfolder
+        )
 
     def build(self):
```

Comment:
The cmake script received extra options:
https://github.com/CrowCpp/crow/blob/b18fbb18f02264f56abf7ebdf664ce3f5f97ece5/CMakeLists.txt#L24-L25

Reply:
I don't think examples and tests are really interesting for CCI? Also, I thought this was a header only lib?

Reply:
Indeed, but if possible we want to disable building tests/examples.

Reply:
Ah I haven't noticed the original recipe did indeed use CMake